LocalSettings.php

```diff
@@ -3395,6 +3402,7 @@
  				'read' => true,
 				'edit' => true,
  			),
+			
```

Comment:
Will it really harm anything? It's so annoying having to scroll through 3000+ lines of code every time. If it absolutely needs to be removed, I'll ask someone else to do it for me.